### DISCUSSION TITLE
Block cracked users with premium usernames.

### DISCUSSION THREAD

**Discussion by DiaxuDev** [games647/FastLogin#817] [Q&A] [answered]
Is there any way to block cracked users trying to join with premium username?

**Comment by ApliNi** [comment, marked as answer]
This does not need to be configured, and offline players will naturally not be able to join the server with any advanced user names, and they will encounter invalid sessions.
Edit: Need to enable `autoRegister`

**Reply by DiaxuDev** [reply]
I will try it, thanks!
